test/parallel/test-crypto-keygen.js

```diff
@@ -123,10 +123,18 @@ const sec1EncExp = (cipher) => getRegExpForPEM('EC PRIVATE KEY', cipher);
   assert.strictEqual(typeof publicKey, 'object');
   assert.strictEqual(publicKey.type, 'public');
   assert.strictEqual(publicKey.asymmetricKeyType, 'rsa');
+  assert.deepStrictEqual(publicKey.asymmetricKeyDetails, {
+    modulusLength: 512,
+    publicExponent: 65537n
+  });
```

Comment:
Nit: Could you add a test for the non-default public exponent (e.g., add this check to the test a few lines above that uses `publicExponent: 3`)?

Reply:
That test uses public and private key encoding, i would have to duplicate that test without key encodings to end up with a key object.

Reply:
added a new test block